src/backend/marsha/core/serializers/video.py

```diff
@@ -380,7 +383,7 @@ def validate(self, attrs):
 
         if not attrs.get("video_id") and isinstance(user, TokenUser):
             attrs["video_id"] = user.id
-
+            attrs["is_registered"] = True
```

Comment:
This should be added in the `create` function of this serializer. You add it to the `validated_data`. This is the only case I think we want to set it to true. WDYT ?

Reply:
In my idea, I wanted to keep this serializer for registrations and use the other one  liveattendance's serializer for the rest, setting it here made this option secure, wdyt ?

Reply:
Yes I understood, you split in two separated serializer and I agree with this. But in this special case, forcing `is_register` to true is only related to the creation of this resource. Adding it all the time to the serializer attribute is I think wrong and should be done only in the creation process. For all other actions, the value should not be modified.